tests/common/nebula_service.py

```diff
@@ -100,6 +100,8 @@ def start(self, debug_log = True):
                                                   metad_ports[0],
                                                   ports,
                                                   debug_log)
+            if server_name == 'graphd':
+                command += ' --enable_authorize=true'
```

Comment:
why add it directly? you can update it by config. don't use it default.

Reply:
Run all tests in parallel.